platform/security/expansion/security-expansion-impl/src/main/java/ddf/security/expansion/impl/AbstractExpansion.java

```diff
@@ -455,7 +457,8 @@ protected void loadConfiguration(String filename) {
         }
         File file = null;
         filename = StringUtils.strip(filename);
-        if (!StringUtils.startsWith(filename, "/") && !StringUtils.startsWith(filename, "\\")) {
+        Path filePath = Paths.get(filename);
+        if (!filePath.isAbsolute()) {
```

Comment:
Not really an issue, but you could do this all in one line `Paths.get(filename).isAbsolute()` since the `filePath` variable isn't getting used anywhere else.